include/bout/invert/laplacexz.hxx

```diff
@@ -38,21 +38,29 @@
 
 class LaplaceXZ {
 public:
-  LaplaceXZ(Mesh *UNUSED(m), Options *UNUSED(options), const CELL_LOC UNUSED(loc)) {}
+  LaplaceXZ(Mesh* m = nullptr, Options* UNUSED(options) = nullptr,
+      const CELL_LOC loc = CELL_CENTRE)
+  : localmesh(m), location(loc) {
```

Comment:
This could use `localmesh(m==nullptr ? mesh : m)` in the initialiser list to leave the function body as empty.

Reply:
Yes, definitely. Sorry, I was being dumb!